app/src/main/java/com/marvelapp/presentation/activity/MainMenuActivity.kt

```diff
@@ -31,16 +32,23 @@ class MainMenuActivity : AppCompatActivity() {
         lifecycleScope.launchWhenStarted {
             viewModel.state.collectLatest { this@MainMenuActivity.updateUI(it) }
         }
+        binding.mainMenuButton.setOnClickListener { viewModel.buttonPressed() }
     }
 
     private fun updateUI(state: MainMenuViewModel.MainMenuStates) {
         when (state) {
             INIT -> {
                 // DO NOTHING
```

Comment:
why is this here?

Reply:
as it is for the default value needed by the stateFlow, you can use it to fall Glide here

Reply:
this still here

Reply:
Solved